app/views/submissions/show.html.erb

```diff
@@ -23,7 +23,7 @@
 <%= " posted @ #{@submission.created_at} by" %> <strong> <%= "#{@submission.user.username}" %> </strong>
 
 <h4>Description</h4>
-<p><%= @submission.description %></p>
+<p><%= @submission.comments[0].body %></p>
```

Comment:
I would save this to an instance variable @description in the controller and do this logic there.